src/styles/blocks/card.scss

```diff
@@ -0,0 +1,85 @@
+.card {
+  width: 200px;
+  padding: 29px 14px 14px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: $standard-color;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__image {
+    height: 134px;
+    width: 160px;
+    margin-bottom: 45px;
+  }
+
+  &__name {
+    margin-bottom: 6px;
+    font-weight: $fw-semibold;
+    font-size: $fs-md;
+    line-height: 15px;
+  }
+
+  &__code {
+    margin-bottom: 15px;
+    align-self: flex-start;
+    font-size: $fs-sm;
+    line-height: 13px;
+  }
+
+  &__rating {
+    width: 100%;
+    margin-bottom: 20px;
+    display: flex;
+    justify-content: space-between;
+  }
+
+  &__feedback {
+    display: flex;
+    align-items: center;
```

Comment:
why do you need these properties here, on flex child? just add align-items: center to the flex parent container